lib/uo_cli.rb

```diff
@@ -655,13 +667,29 @@ def self.check_reader
       end
 
       # We're calling the python cli that gets installed when the user installs ditto-reader.
-      # If ditto-reader is installed into a venv (recommended), that venv must be activated for this to work.
-      # Haaaaaaaacky!
-      begin
-        system("ditto_reader_cli run-opendss -s #{@opthash.subopts[:scenario]} -f #{@opthash.subopts[:feature]}")
-      rescue FileNotFoundError
-        abort("\nMust post-process results before running opendss. We recommend --default." \
-        "Once opendss is run, you may then 'process --opendss'")
+      # If ditto-reader is installed into a venv (recommended), that venv must be activated when this command is called.
+      if @opthash.subopts[:scenario] && @opthash.subopts[:feature]
```

Comment:
@kflemin I can't pass a flag to the python cli if it hasn't been mentioned in the ruby cli. I could make a big nest with all options individually listed, but that feels icky. Do you have a recommendation for a better way?